crates/amaru/tests/summary.rs

```diff
@@ -65,50 +65,42 @@ fn db(epoch: Epoch) -> Arc<impl Snapshot + Send + Sync> {
     handle
 }
 
-#[test_case(163)]
-#[test_case(164)]
-#[test_case(165)]
-#[test_case(166)]
-#[test_case(167)]
-#[test_case(168)]
-#[test_case(169)]
-#[test_case(170)]
-#[test_case(171)]
-#[test_case(172)]
-#[test_case(173)]
-#[test_case(174)]
-#[test_case(175)]
-#[test_case(176)]
-#[test_case(177)]
-#[test_case(178)]
-#[test_case(179)]
-#[ignore]
+include!(concat!(
```

Comment:
I like having explicit list of test cases, could we keep it but add a `NetworkName` parameter? Also, not a great fan of relying on environment variable to run tests.

Reply:
The `#[ignore]` annotation was dropped, is that intended? Since those snapshots test require to have already run the node, we need to exclude them from standard unit tests.

Reply:
Ah no, I see, the test cases have been moved to be generated by the `build.rs`